python/nav/models/manage.py

```diff
@@ -154,8 +154,13 @@ def is_snmp(self):
 
     @property
     def snmp_version(self):
+        """Returns the configured SNMP version as an integer"""
         if self.is_snmp:
-            return self.configuration['version']
+            value = self.configuration['version']
+            if value or value == 0:
+                if value == "2c":
+                    return 2
+            return int(value)
```

Comment:
This is no different than this:
```suggestion
            if value == "2c":
                return 2
            return int(value)
```

So why the redundant `if` statement?

Reply:
Very good question :sweat_smile: